workflows/treeval.nf

```diff
@@ -133,7 +140,7 @@ workflow TREEVAL {
     // SUBWORKFLOW: Takes reference, channel of enzymes, my.genome, assembly_id and as file to generate
     //              file with enzymatic digest sites.
     //
-    if ( !(exclude_workflow_steps?.contains("insilico_digest"))) {
+    if ( include_workflow_steps?.contains("insilico_digest")) {
```

Comment:
```suggestion
    if ( include_workflow_steps.contains("insilico_digest")) {
```
Why would `include_workflow_steps` be `null`? More below.

Reply:
I wouldn't be now, but could have been in older versions.

Removed now.